decode/src/datapath.rs

```diff
@@ -194,7 +194,20 @@ impl schemars::JsonSchema for EthernetComplianceCode {
     }
 
     fn json_schema(gen: &mut schemars::gen::SchemaGenerator) -> schemars::schema::Schema {
-        String::json_schema(gen)
+        let mut s = String::json_schema(gen);
+        let ::schemars::schema::Schema::Object(obj) = &mut s else {
+            unreachable!();
+        };
+        obj.metadata
```

Comment:
https://docs.rs/schemars/latest/schemars/schema/struct.SchemaObject.html#method.metadata does what you want here

Reply:
Nice, thanks, I'm using that in a038a0b